setup.py

```diff
@@ -17,7 +50,9 @@
     "botocore==1.12.220",  # Apache License 2.0
     "boto3==1.9.220",  # Apache License 2.0
     "awscli==1.16.230",  # Apache License 2.0
-    "faker==2.0.0",  # MIT License
```

Comment:
need to fix `faker` dependency. it seems to have been moved from`dev_require` to `install_requires` in master